### PR TITLE
Feature/pp9 1090 layers align cp

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts [hunk 1]
```diff
@@ -85,15 +96,29 @@ export class LayerPanelsComponent implements OnInit {
...
         });
+        bottomLayers.forEach((bottomLayer) => this.layers.push(bottomLayer));
       });
   }
 
   public relationClickHandler(relationInfo: RelationFieldInfo) {
     this.relationClick.emit(relationInfo);
   }
 
+  public layerExpansionHandlder(layerId: string, opened: boolean) {
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts:113]: Fix typo

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts [hunk 2]
```diff
@@ -28,11 +28,19 @@ export class LayerPanelsComponent implements OnInit {
...
+  public expandedSchemas: Set<string> = new Set<string>();
+
   private allSchemas: SchemaDetail[];
 
+  localStorageKey = 'SchemaExpansionState';
+
   constructor(private schemaService: SchemaService, private layerFieldService: LayerFieldService) {}
 
   ngOnInit() {
+    const expandedSchemasString = localStorage.getItem(this.localStorageKey);
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts:40]: Use the localStorage service

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts (hunk 1 above)
Comment (rjs-picturepark) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts:113]: Done

File: src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/app.component.ts [hunk 3]
```diff
@@ -13,16 +13,21 @@ export class AppComponent implements OnInit {
   public shareDetail: ShareDetail;
   public mailRecipients: IMailRecipient[];
   public showSearchBox = !environment.production;
-  public showConsent = !window.localStorage.getItem('Picturepark.ShareCookieConsent');
+  public showConsent = false;
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/app.component.ts:16]: I think this should be true by default

**Pull request merged**